packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -464,13 +461,19 @@ class FileCollection extends DocumentCollection {
    * @param {string} id   - The file's id
    * @param {string} [name]   - The file copy name
    * @param {string} [dirId]   - The destination directory id
+   * @param {object} [options] - Optionnal request options
    * @returns {Promise<object>}   - A promise that returns the copied file if resolved.
    * @throws {FetchError}
    *
    */
-  async copy(id, name, dirId) {
+  async copy(id, name, dirId, { sanitizeName = true } = {}) {
+    let sanitizedName = name
+    if (sanitizedName != null && sanitizeName) {
```

Comment:
You slightly changed the condition here, from `=== undefined` to `!= null`. Is there a reason ?

Reply:
Ok, thanks